src/components/User.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+function User({ user }) {
+  return (
+    <>
+      {user && (
+        <div className="userCard">
+          <img src={user.picture.medium} alt="User" />
```

Comment:
Alt should be empty here or "Profile picture." or something; just "User" has no added value.
```suggestion
          <img src={user.picture.medium} alt="" />
```